.github/workflows/test.yml

```diff
@@ -1,70 +1,69 @@
 name: test-run
```

Comment:
> previously, all jobs worked only against Python 2.7

I printed python version from test-run and [pushed](https://github.com/tarantool/test-run/commit/32aba8a686659193f816069beed9a7de41ab3861) it to trigger CI. The patch:

```diff
diff --git a/test-run.py b/test-run.py
index f706d4b..5e836be 100755
--- a/test-run.py
+++ b/test-run.py
@@ -291,6 +291,9 @@ if __name__ == "__main__":
 
     status = 0
 
+    color_stdout('sys.version: {}\n'.format(sys.version))
+    color_stdout('sys.executable: {}\n'.format(sys.executable))
+
     if Options().args.show_tags:
         show_tags()
         exit(status)
```

The output from the `build (2.7, 2.7)` job:

```
sys.version: 2.7.18 (default, Oct 29 2020, 08:03:56) 
[GCC 9.3.0]
sys.executable: /opt/hostedtoolcache/Python/2.7.18/x64/bin/python
```

The output from the `build (3.5, 2.7)` job:

```
sys.version: 3.5.10 (default, Nov 25 2020, 13:18:55) 
[GCC 9.3.0]
sys.executable: /opt/hostedtoolcache/Python/3.5.10/x64/bin/python
```

I see that that the step, which installs dependencies for Python 2, is not skipped on Python 3. However the statement that we test only Python 2 is not correct.

Do you want more strange things to look into?

* It seems, in python 3 jobs we install dependencies for python 2, but everything works. I guess the installed python bundle already have necessary dependencies.

Please, if you'll dig into all those things, fix them step by step, not everything in one commit. I hope it is not so hard work.

----

Wow, fresh gevent [supports](http://www.gevent.org/install.html#supported-platforms) latest python 2.7. I had a guess that out `requirements.txt` works only for Python 3, but it works for both. Nice.

Reply:
Now the PR has no github workflow changes, so I'll resolve this discussion.